src/main/java/com/movie_collection/be/Movie.java

```diff
@@ -0,0 +1,7 @@
+package com.movie_collection.be;
```

Comment:
love that we are using record :) but I believe niko suggested using StringProperty or smith since its will be easier to map into table
but also depends if we decide to use DTO's

Reply:
Implemented here

https://github.com/TomassSimko/PrivateMovieCollection/pull/7/commits/47c07bba107e9f3ca671e99a4884d35de9c9122d